idaes/tests/performance/performance_test_base.py

```diff
@@ -0,0 +1,149 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Author: Andrew Lee
+"""
+import gc
+
+import pyomo.common.unittest as unittest
+from pyomo.util.check_units import assert_units_consistent
+from pyomo.common.timing import TicTocTimer
+from pyomo.environ import assert_optimal_termination
+
+from idaes.core.solvers import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+# Get default solver for testing
+solver = get_solver()
+
+
+class PerformanceBaseClass:  # unittest.TestCase):
```

Comment:
Inheriting from TestCase here causes this base class to show up when using `unittest.TestLoader.discover`, so I think it should not happen (and the comment can be removed). Presumably we could do discovery without unittest, but I have not figured it out.

Reply:
Yes - this is why this was commented out. I just hadn't cleaned it up yet.